.github/workflows/CI.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 4 * * MON'
+
+jobs:
+  CI:
+    strategy:
+      matrix:
+        backend: ["SERIAL", "OPENMP"]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install deps
+        run: sudo apt-get update && sudo apt-get install openmpi-bin libopenmpi-dev
+      - name: Checkout kokkos
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: kokkos/kokkos
+          ref: 3.4.01
+          path: kokkos
+      - name: Build kokkos
+        working-directory: kokkos
+        run: |
+          cmake -B build \
+            -DCMAKE_INSTALL_PREFIX=$HOME/kokkos \
+            -DKokkos_CXX_STANDARD=14 \
+            -DKokkos_ENABLE_${{ matrix.backend }}=ON
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Checkout Cabana
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: ECP-copa/Cabana
+          ref: master
+          path: Cabana
+      - name: Build Cabana
+        working-directory: Cabana
+        run: |
+          cmake -B build \
+          -DCMAKE_INSTALL_PREFIX=$HOME/Cabana \
+          -DCMAKE_PREFIX_PATH="$HOME/kokkos" \
+          -DCabana_REQUIRE_${{ matrix.backend }}=ON
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Checkout code
+        uses: actions/checkout@v2.2.0
+      - name: Build
+        run: |
+          cmake -B build -DCMAKE_PREFIX_PATH="$HOME/Cabana"
+          cmake --build build --parallel 2
```

Comment:
No tests?

Reply:
I wasn't sure how to reduce the test size. @steverangel if you add an input file that runs in ~seconds in serial we could run here